rllib/BUILD

```diff
@@ -1190,18 +1189,26 @@ py_test(
 )
 
 py_test(
-    name = "examples/multiagent_cartpole",
+    name = "examples/multi_agent_cartpole",
     tags = ["examples", "examples_M"],
     size = "medium",
-    srcs = ["examples/multiagent_cartpole.py"],
+    srcs = ["examples/multi_agent_cartpole.py"],
     args = ["--num-iters=2", "--num-cpus=4"]
 )
 
+# Broken (NOTE: this test was not in Jenkins: run_rllib_tests.sh)
```

Comment:
Just tried this out and it can be fixed with this: 
```
diff --git a/rllib/utils/sgd.py b/rllib/utils/sgd.py
index 04ce2b662..5c23c9979 100644
--- a/rllib/utils/sgd.py
+++ b/rllib/utils/sgd.py
@@ -109,7 +109,7 @@ def do_minibatch_sgd(samples, policies, local_worker, num_sgd_iter,
                     MultiAgentBatch({
                         policy_id: minibatch
                     }, minibatch.count)))[policy_id]
-                for k, v in batch_fetches[LEARNER_STATS_KEY].items():
+                for k, v in batch_fetches.get(LEARNER_STATS_KEY, {}).items():
```

Feel free to roll this into this PR, otherwise I can fix it separately.